### ISSUE TITLE
OMOP CDM V5.4 support ?

### ISSUE THREAD

**Issue by Volpym** [OHDSI/ArachneExecutionEngine#367]
Are there any plans to support OMOP CDM v5.4?

**Comment by konstjar** [comment]
Execution Engine does not know anything about OMOP CDM version. It should be added in the client application. Can you please specify what application do you use for submitting analysis in Execution Engine?

**Comment by Volpym** [comment]
I should have provided more context for my question. 
I have some synthetic data which I converted to OMOP CDM using Synthea-ETL. The CDM Version that I am using is V5.4. 

Running Achilles manually works perfectly. Trying to generate the statistics through Arachne's Datanode UI is not working and I thought that it had to do with the omop cdm version. The reason I posted my question here is because to my understanding EE is responsible for downloading the achilles docker image and running it. Did I misunderstood something ?
